templates/leptos-ssr/content/src/app.rs

```diff
@@ -1,35 +1,57 @@
-use leptos::*;
+use components::{Route, Router, Routes};
+use leptos::{prelude::*, task::spawn_local};
 use leptos_meta::*;
 use leptos_router::*;
 
+#[cfg(feature = "ssr")]
+pub fn shell(options: LeptosOptions) -> impl IntoView {
+    view! {
+        <!DOCTYPE html>
+        <html lang="en">
+            <head>
+                <AutoReload options=options.clone() />
+                <HydrationScripts options=options.clone()/>
+                <MetaTags/>
+                <Stylesheet id="leptos" href="/pkg/{{project-name | snake_case}}.css"/>
+            </head>
+            <body>
+                <App/>
+            </body>
+        </html>
+    }
+}
+
 #[component]
 pub fn App() -> impl IntoView {
     // Provides context that manages stylesheets, titles, meta tags, etc.
     provide_meta_context();
 
+    let fallback = || view! { "Page not found." }.into_view();
+
     view! {
-      <Stylesheet id="leptos" href="/pkg/{{project-name | snake_case}}.css"/>
+        <Meta name="charset" content="UTF-8"/>
+        <Meta name="description" content="A website running its server-side as a WASI Component :D"/>
+        <Meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <Meta name="theme-color" content="white"/>
 
-      // sets the document title
-      <Title text="Welcome to Leptos"/>
+        <Title text="Welcome to Leptos X WebAssembly!"/>
```

Comment:
If we wish to change this I would suggest Spin rather than WebAssembly (because Leptos uses WebAssembly on the client anyway).

Reply:
Ahh, this was some copy pasta from merging the templates. Will update now!